ddg-idd-driver (1.0.12) stable; urgency=medium


[Sergey Zhukov]
* 1.0.12 - Wrong dependency was removed
[Sergey Zhukov]
* 1.0.11 - New install procedure
[Sergey Zhukov]
* 1.0.10 - Configuration file
[Vladimir Stativa]
* 1.0.9 - compatible with Python3.5.3
[Vladimir Stativa]
* 1.0.8 - fixed DDG error 
[Vladimir Stativa]
* 1.0.7 - added IDD 
[Vladimir Stativa]
* 1.0.6 - int value = correct

